src/data/__tests__/calculate.test.ts

```diff
@@ -258,10 +260,11 @@ describe('calculate', () => {
       expect(calcValue(partner)).toEqual(calcValue(bonuses))
     })
 
-    it('should apply 48% risk', () => {
+    it('should apply 75% risk', () => {
```

Comment:
Would it be more accurate to put 72% here?

Reply:
updated